README.md

```diff
@@ -0,0 +1,18 @@
+# minithulhu
+
+An example for DigitalOcean's monorepo as described in:
+
+* [https://blog.gopheracademy.com/advent-2015/go-in-a-monorepo/](https://blog.gopheracademy.com/advent-2015/go-in-a-monorepo/)
+* [https://www.digitalocean.com/company/blog/taming-your-go-dependencies/](https://www.digitalocean.com/company/blog/taming-your-go-dependencies/)
+
+## Vendoring new code
+
+If you wish to add a third party dependency to the repo, put the code you
```

Comment:
Want to add the note that `go get github.com/my/dependency` would put the code in the correct place and fetch all deps of that dep?